lib/kahuna_client/client/server.rb

```diff
@@ -12,15 +12,26 @@ module Server
       # event (e.g. event = start) Event Name
       # user_info (e.g. user_info = {'first_name': 'John', 'last_name': 'Doe', 'gender': 'm'})
       def send_event(options = {})
+        events = if options[:events]
+                       options[:events].each do |key, obj|
```

Comment:
I'll need to address this once my question above is answered. 
If there is an events parameter all the properties in it need to be an array. 

I'm not worried so much about the logic right now. I'm more concerned with the answer to the question above which will resolve any issues here.